src/color/setting.js

```diff
@@ -827,15 +1333,16 @@ p5.prototype.noStroke = function() {
 };
 
 /**
- * Sets the color used to draw lines and borders around shapes. Calling
- * `stroke(255, 165, 0)` or `stroke('orange')` means all shapes drawn after
- * the `stroke()` command will be filled with the color orange. The way these
- * parameters are interpreted may be changed with the
+ * Sets the color used to draw points, lines, and the outlines of shapes.
```

Comment:
Same here. How about 'Sets the color used to draw the outlines of shapes'?
Not sure if we need 'points' and 'lines' here. @limzykenneth @davepagurek